store/src/meta_service/raftmeta_test.rs

```diff
@@ -381,6 +381,7 @@ async fn test_meta_node_add_database() -> anyhow::Result<()> {
 }
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 10)]
+#[flaky_test]
```

Comment:
Hi @zhyass ,
@drmingdrmer is working on it, so it's better to add it in another patch.

Reply:
ok